Packs/FeedThreatConnect/Integrations/FeedThreatConnect/FeedThreatConnect.py

```diff
@@ -191,12 +183,56 @@ def fetch_indicators_command(client: Client) -> List[Dict[str, Any]]:
     Returns:
         list: indicator to populate in demisto server.
     """
-    raw_response = client.get_indicators(owners=argToList(demisto.getParam('owners')))
+    owners = f'AND ({create_or_query(demisto.getParam("owners"), "ownerName")}) '
+    tags = f'AND ({create_or_query(demisto.getParam("tags"), "tags")}) '
+    status = f'AND ({create_or_query(demisto.getParam("status"), "status")}) '
+    fields = set_fields_query(argToList(demisto.getParam("fields")))
+    group_type = f'AND ({create_or_query(demisto.params().get("group_type", "Incident"), "typeName")}) '
+    first_fetch_time = demisto.getParam("first_fetch_time")
+    from_date = ''
+    if first_fetch_time:
+        first_fetch_time = dateparser.parse(demisto.getParam('first_fetch_time').strip()).strftime("%Y-%m-%d")  # type: ignore # noqa
+        from_date = f'AND (dateAdded > "{first_fetch_time}") '
+    page = 0
+    tql = f'{owners if owners != "AND () " else ""}{tags if tags != "AND () " else ""}' \
+          f'{group_type if group_type != "AND () " else ""}{status if status != "AND () " else ""}' \
+          f'{from_date if from_date != "AND () " else ""}'.replace('AND', '', 1)
+    if tql:
+        tql = urllib.parse.quote(tql.encode('utf8'))  # type: ignore
+        tql = f'?tql={tql}'
+    else:
+        tql = ''
+        if fields:
+            fields = fields.replace('&', '?', 1)  # type: ignore
+    url = f'/api/v3/groups{tql}{fields}&resultStart={page}&resultLimit=500'
```

Comment:
Do not pass query parameters as part of the URL; There is a dedicated argument for it.